Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -2005,15 +2059,22 @@ def slack_send_file(_channel: str | None = None, _channel_id: str = '', _entry_i
         'name': file_path['name'],
         'comment': comment
     }
+
+    error_message = 'Could not send the file to Slack.'
```

Comment:
can we add some information about the file, like its name, to the error message? Will make it easier for users to understand what went wrong

Reply:
I have now added the file name in the error message in commit 86a9eb17e04ab27552e682f68301ea4abf685423.
Note that the explanation of the error is concatenated with the error message under the `except ...` block a few lines below